Okta.Xamarin/Okta.Xamarin/OktaStateManager.cs

```diff
@@ -4,19 +4,24 @@
 // </copyright>
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
 using System.Threading.Tasks;
 
 namespace Okta.Xamarin
 {
     /// <summary>
     /// Tracks the current login state, including any access tokens, refresh tokens, scope, etc.
     /// </summary>
-    public class OktaState
+    public class OktaStateManager
     {
+        private HttpClient client = new HttpClient();
+
         /// <summary>
         /// Gets the token type.
         /// </summary>
-        public string TokenType { get; private set; }
+        public string TokenType { get; private set; } // TODO: delete this
```

Comment:
Is this TODO still relevant?

Reply:
Yes, it isn't used but if I deleted it I would have to review a lot of code that references it, mostly code that calls the constructor.  I'll handle that in a separate PR.